docs/src/main/sphinx/release/release-389.md

```diff
@@ -0,0 +1,40 @@
+# Release 389 (6 July 2022)
+
+## General
+
+* Improve performance of queries which contain a large number of aggregations
+  that use `RowType` for representing intermediate state (e.g. `SUM`). ({issue}`12762`)
+* Remove deprecated version of `ConnectorRecordSetProvider#getRecordSet`. ({issue}`13084`)
+* Fix query failure when no arguments are passed to a table function. ({issue}`12951`)
+* Fix error where a split keeps running after query failure due to an
+  uninterruptible regular expression function. ({issue}`13064`)
+
+## Delta Lake connector
+
+* Improve native parquet writer performance for flat types. ({issue}`13030`)
```

Comment:
I suppose users won't understand what "flat types" is.

Reply:
Maybe "primitive" or "non-nested" types then ?